test/helpers/index.ts

```diff
@@ -1,30 +1,32 @@
-import { BreadcrumbsConfiguration } from "src/config/interfaces";
-import { parseInt } from "src/utils/parse";
+import {
+  BreadcrumbsConfiguration,
+  BreadcrumbsNetworkConfiguration,
+} from "src/config/interfaces";
+import { parseInt, parseFloat } from "src/utils/parse";
 
 const DEFAULT_BAKER = "tz1Uoy4PdQDDiHRRec77pJEQJ21tSyksarur";
+const { PKEY } = process.env;
 
 export const generateConfig = (
-  args: {
-    baking_address?: string;
-    overdelegation_guard?: boolean;
-    overdelegation_blacklist?: string[];
-    redirect_payments?: { [key: string]: string };
-    fee_exceptions?: { [key: string]: string };
-    default_fee?: string;
-    minimum_payment_amount?: string;
-    minimum_delegator_balance?: string;
-    fee_income_recipients?: { [key: string]: string };
-    bond_reward_recipients?: { [key: string]: string };
-  } = {}
+  args: Partial<BreadcrumbsConfiguration> = {}
 ): BreadcrumbsConfiguration => {
   return {
     baking_address: args.baking_address || DEFAULT_BAKER,
+    payout_wallet_key: args.payout_wallet_key || PKEY || "",
```

Comment:
I know this will be moved to another file but I would specify that it is the private key in any naming.